certification/internal/shell/has_prohibited_packages_test.go

```diff
@@ -0,0 +1,40 @@
+package shell
```

Comment:
I think we better have the tests in a separate directory. Probably in `certification/internal/shell/test`

Reply:
So I have been working on this and might be a little harder than simply moving to a new folder. These tests use the the validate (lower case) so it is not an exported function. When the tests are moved to a different package it can't access that function. So I guess there are two options.

1) Move tests to their own package. Rename validate to something else so it can be an exportable function.
2) Keep it as is and have tests be in the same package as the checks

I can definitely do first option just want to get the okay because I would have to update all of the checks.

Reply:
Here is my thought (I am new to Ginkgo so please take this with grains of salt):

I suggest containing the PodmanEngine creation in a method (say `Engine()`), and merging the two functions (validate and Validate). We can then mock the call to `Engine()` function by returning a `FakePodman` instance. In this case, we test the Validate() method (which is the public interface of checks), we ended up with fewer LoCs and we can move tests to a separate package.

Reply:
1) This needs to be moved to the new repo
2) Take a look at redhat-openshift-ecosystem/openshift-preflight#1